docs2/site/docs/getting-started/errors.md

```diff
@@ -200,3 +200,36 @@ For example, to show the stack traces for unhandled errors during development, y
 You can also write your own implementation of `IErrorInfoProvider`. For instance, you might want to override
 the numerical codes provided by GraphQL.NET for validation errors, or reveal stack traces
 only to logged-in administrators.
+
+## <a name="ValidationErrors"></a>Validation error reference list
+
+Here is a full list of validation errors produced by GraphQL.NET:
```

Comment:
Let's order this list by error number, not by rule name/code.

Reply:
Done